README.md

```diff
@@ -41,4 +41,7 @@ See HELP.md and LICENSE
 * Add ID option for feedback colors
 
 **V1.1.6**
-* Prevent re-initializing variables if module is being disabled
\ No newline at end of file
+* Prevent re-initializing variables if module is being disabled
```

Comment:
```suggestion
* Prevent re-initializing variables if module is disabled
```

Reply:
The comment syntax is correct. The variables were being re-initialized _during_ the disable/destroy process. 
They are now prevented from being re-created when the module is _being_ disabled, but the process is not finished.

Reply:
Can I suggest  that it is therefore
```suggestion
* Prevent re-initializing variables while the module is being disabled
```

Reply:
That one is acceptable. The patch being described is a small, specific change, so I like for that comment to be more precise without torturing the language/shorthand too much.